### ISSUE TITLE
Support Rtl

### ISSUE THREAD

**Issue by erfjab** [nisabmohd/Aria-Docs#9]
Hi, does it support languages like Persian?

**Comment by nisabmohd** [comment]
Good idea... I can make an future version which supports `i18n` for multiple languages

**Comment by fotisskal** [comment]
Hello! Is there any progress maybe regarding i18n support?

**Comment by nisabmohd** [comment]
![Image](https://github.com/user-attachments/assets/87194a2b-6f7a-40e3-ae2e-f95ee3a07d51)

![Image](https://github.com/user-attachments/assets/ce63ea35-d68b-4f41-97f4-a20ce69cc78c)

![Image](https://github.com/user-attachments/assets/6e9d28e9-d2c4-4b5a-b332-9c82b377d5e0)

**Comment by nisabmohd** [comment]
Added the feature ✌️
please raise issue if any

**Comment by nisabmohd** [comment]
Please find this on branch
https://github.com/nisabmohd/Aria-Docs/tree/i18n-support
